### PR TITLE
Creates new sidebar component

### PR DESCRIPTION
**Pull request by mattlaguardia**
# Description

This PR creates the Sidebar component to be used in our Layout component. Right now some cards are hard coded... will address that at a later date as well as mobile. 

Screen Shot 2020-11-24 at 10 40 04 AM

### CODE REVIEW

File: src/assets/purple-blast.svg [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<svg width="41" height="41" viewBox="0 0 41 41" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (mcheng09) [inline src/assets/purple-blast.svg:1]: These SVGs are images and not paths. Will we have the path versions of it?

File: src/components/Layout/Layout.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import Sidebar from 'components/Sidebar/Sidebar';
 
 const Layout = ({ noNav, children }) => {
   if (noNav) {
```
Comment (mcheng09) [inline src/components/Layout/Layout.js:5]: Was wondering if you can update `noNav` to `noSideBar`. Might be clearer.

**Comment by mcheng09** [comment]
Screen Shot 2020-11-27 at 3 54 18 PM

Requested Fixes: 
Title font-weight.
Learn More font-weight.
Learn More lowercase. 
Set border-radius to 4px for cards.

**Review by mcheng09 — COMMENTED** [review]
Nice! I left a few comments, feel free to let me know if you have any questions about any of 'em.

File: src/components/Layout/Layout.js (hunk 2 above)
Comment (mattlaguardia) [reply src/components/Layout/Layout.js:5]: Lets not comment on this in my PR, instead lets open an issue so we can address it in a code clean up later

Comment (mattlaguardia) [reply src/components/Layout/Layout.js:5]: But yea good call thats a better name

File: src/assets/purple-blast.svg (hunk 1 above)
Comment (mattlaguardia) [reply src/assets/purple-blast.svg:1]: No but lets not worry about that for now design needs to export them

Comment (mattlaguardia) [reply src/assets/purple-blast.svg:1]: But yes they are images and hopefully im not sure whats the status of them waiting on design but dont want to hang on this

**Pull request merged**